### PR TITLE
Aggregation

### PR DESCRIPTION
**Pull request by tomxi**
Implements the different feature aggregation methods in the `data_aggr.py` module

### CODE REVIEW

File: sonyc_kalman/data.py [hunk 1]
```diff
@@ -73,13 +75,30 @@ def load_openl3_time_series(hdf5_path, delta_mins=15, aggr_func=None):
...
+        aggr_func = data_aggr.random
+    if args.centroid:
+        aggr_func = data_aggr.centroid
+    if args.medoid:
+        aggr_func = data_aggr.medoid
+    if args.anti_medoid:
+        aggr_func = data_aggr.anti_medoid
+        
 
     out_fname = "{}_{}minslot.npz".format(os.path.basename(args.hdf5_path).split('.')[0],
```
Comment (jtcramer) [inline sonyc_kalman/data.py]: We should add the aggregation method in the filename

**Review by jtcramer — CHANGES_REQUESTED** [review]
Just the filename needs to be changed. Otherwise LGTM.

**Comment by ChrisIck** [comment]
LGTM, no issues other than the naming conventions @jtcramer pointed out.

**Comment by jtcramer** [comment]
LGTM

**Review by jtcramer — APPROVED** [review]

**Pull request merged**